nomos-core/chain-defs/src/blend/mod.rs

```diff
@@ -6,3 +9,22 @@ pub const PROOF_CIRCUIT_SIZE: usize = size_of::<PoQProof>();
 pub const PROOF_OF_QUOTA_SIZE: usize = KEY_NULLIFIER_SIZE.checked_add(PROOF_CIRCUIT_SIZE).unwrap();
 
 pub const PROOF_OF_SELECTION_SIZE: usize = size_of::<ZkHash>();
+
+#[must_use]
+pub fn core_quota(
+    rounds_per_session: NonZeroU64,
+    message_frequency_per_round: NonNegativeF64,
+    num_blend_layers: NonZeroU64,
+    membership_size: usize,
+) -> u64 {
+    // `C`: Expected number of cover messages that are generated during a session by
+    // the core nodes.
+    let expected_number_of_session_messages =
+        rounds_per_session.get() as f64 * message_frequency_per_round.get();
+
+    // `Q_c`: Messaging allowance that can be used by a core node during a single
+    // session. We assume `R_c` to be `0` for now, hence `Q_c = ceil(C * (ß_c
+    // + 0 * ß_c)) / N = ceil(C * ß_c) / N`.
```

Comment:
why assuming 0?

Reply:
We do not use this parameter at the moment, at least not in Blend. The formula includes it, but then we say this is not used for v1, so since if this were to be different than `0` we would still not do anything with it, we removed from our formula implementation and use `0` instead. Cc @madxor

Reply:
Yes. V1 will not really use this parameter, it's only defined but assumed to be `0`. Hence we simplified the code by removing a parameter we were moving around without really using it, even if it was given any other value than `0`.